website/docs/r/iam_trusted_profiles_claim_rule.html.markdown

```diff
@@ -0,0 +1,62 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_iam_trusted_profile_claim_rule"
+description: |-
+  Manages iam_trusted_profile_claim_rule.
+subcategory: "IAM Identity Services"
+---
+
+# ibm_iam_trusted_profile_claim_rule
+
+Create, update, or delete an IAM trusted profiles claim rule resource. For more information, about IAM trusted profiles claim rule, see https://cloud.ibm.com/apidocs/iam-identity-token-api#create-claim-rule
```

Comment:
Create, update, or delete an IAM trusted profiles claim rule resource. For more information, about IAM trusted profiles claim rule, see https://cloud.ibm.com/apidocs/iam-identity-token-api#create-claim-rule

as

Create, update, or delete an IAM trusted profiles claim rule resource. For more information, about IAM trusted profile claim rule, see [Create claim rule for a trusted profile](https://cloud.ibm.com/apidocs/iam-identity-token-api#create-claim-rule)